pages/index.vue

```diff
@@ -34,11 +34,11 @@ const { data: workshops } = useAsyncData('latest-workshops', () => queryContent(
   <div>
     <AppSection>
       <div class="md:flex md:flex-row items-center">
-        <div class="mt-8 md:mt-0 md:w-3/5 md:flex-1">
+        <div class="mt-8 md:mt-0 md:w-3/5 md:flex-1 md:flex-grow-[1.05] lg:flex-grow-[1.05]">
           <div class="flex gap-8 -mx-4 md:mx-0">
             <div class="flex w-1/2 md:w-full items-center">
               <AppParagraph tag="h1" look="superHeading"
-                class="flex flex-col sm:justify-start font-mono mt-8 md:mt-0 ml-4 md:ml-0">
+                class="flex flex-col sm:justify-start font-mono mt-8 md:mt-0 ml-4 md:ml-0 md:text-[55.1px] lg:text-[83px]">
```

Comment:
Can we use rem's instead of pixels here? Ideally via predefined classes like `text-xl`? 🤔